### ISSUE TITLE
Message context/right click menu

### ISSUE THREAD

**Issue by tezlm** [ajbura/cinny#849]
### Describe the problem

Clicking the buttons on a toolbar can be slow. Deleting a few messages takes longer than feels it should. Plus, having a context menu is nice to have.

### Describe the solution you'd like

Show a menu when right clicking a message, similar to rooms and spaces.

### Alternatives considered

Not having a context menu

### Additional context

element context menu

![image](https://user-images.githubusercontent.com/56274831/190617206-54792a66-e571-4b98-9fbe-64c7b3b9b313.png)

discord context menu

![image](https://user-images.githubusercontent.com/56274831/190617254-118feb19-91f5-43bb-8938-f57777467c2f.png)

(Simply showing the 5 toolbar options as a menu would be enough)

**Comment by Sojens** [comment]
I agree with this, and it is, in my opinion, a MAJOR dealbreaker when trying to use Cinny. I like the UI. I like everything about it. But, this doesn't fit what I need it for. It would not harm anyone, and it is probably an easy adjustment. (I don't know, so sorry if that comment was belittling)

Anyway, I really do hope to see this fixed later!

**Comment by tezlm** [comment]
Added as of #1346 (works in [dev.cinny.in](https://dev.cinny.in)!)

It currently opens the menu at the toolbar position instead of the cursor position (ie always on the right side of the message), but that feels like it should be opened as a bug report issue instead.
